<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FeAture Explorer (Research Used Only)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,2,0,1,0">
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>FeAture Explorer (FAE)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Feature Extraction</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Binay Classification</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Survival Analysis</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonFeatureExtraction">
         <property name="text">
          <string>Feature Extraction</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="buttonBcFeaturePreprocessing">
           <property name="text">
            <string>Feature Preprocessing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBcModelExploration">
           <property name="text">
            <string>Model Exploration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBcVisulization">
           <property name="text">
            <string>Visulization</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonSaModelExploration">
           <property name="text">
            <string>Model Exploration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSaVisulization">
           <property name="text">
            <string>Visulization</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelIntroduction">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Yang Song (songyangmri@gmail.com)
Jing Zhang (zhangjingmri@gmail.com)
Chengxiu Zhang (cxzhang@phy.ecnu.edu.cn)
Guang Yang (gyang@phy.ecnu.edu.cn)</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelIntroduction_2">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Shanghai Key Laboratory of Magnetic Resonance
East China Normal University
3663, Zhongshan North Road. Shanghai, China, 200062</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Song, Yang et al. FeAture Explorer: A tool for developing and comparing radiomics models. PLoS ONE 15 (2020)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
